beacon-chain/core/light-client/lightclient.go

```diff
@@ -551,3 +552,193 @@ func NewLightClientUpdateFromOptimisticUpdate(update *ethpbv2.LightClientOptimis
 		SignatureSlot:  update.SignatureSlot,
 	}
 }
+
+func ComputeTransactionsRoot(payload interfaces.ExecutionData) ([]byte, error) {
```

Comment:
No need to export

Reply:
> No need to export

not exporting leads to function not found in lightclient_test.go